index.html

```diff
@@ -66,5 +66,243 @@ <h2 class="introduction-title">I'm a Sofware Developer</h2>
+            Printing Data
+          </h5>
+          <p class="art-paragraph">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="languages">
+            <li class="programming-boxes second-programming">
+              <a href="#">html</a>
+            </li>
+            <li class="programming-boxes second-programming">
+              <a href="#">bootsrap</a>
+            </li>
+            <li class="programming-boxes second-programming">
+              <a href="#">Ruby</a>
+            </li>
+          </ul>
+        </div>
+        <a class="see-project-button long-button" href="#">See Project</a>
+      </div>
+    </section>
+
+    <section class="about-me-section-main">
+      <img
+        class="background-img2"
+        src="img/shapes-compile.png"
+        alt="shapes compile"
+      />
+      <div class="about-me-section">
+        <h5 class="about-me-heading">About Me</h5>
+        <p class="about-me-paragraph">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <a class="programming-boxes get-my-resume" href="#">Get My Resume</a>
+        <hr />
+        <div class="languages-frame-skills">
+          <div class="heading-box">
+            <img
+              class="rectangular"
+              src="img/Rectangle1.png"
+              alt="rectangular shape"
+            />
+            <h6 class="skills-headings">Languages</h6>
+          </div>
+          <ul>
+            <li class="list-items">Javscript</li>
+            <li class="list-items">Ruby</li>
+            <li class="list-items">Html</li>
+            <li class="list-items">Css</li>
+          </ul>
+          <div class="heading-box">
+            <img
+              class="rectangular"
+              src="img/Rectangle3.png"
+              alt="strght rectangular shape"
+            />
+            <h6 class="skills-headings">Frameworks</h6>
+          </div>
+          <ul>
+            <li class="list-items">Bootsrap</li>
+            <li class="list-items">Ruby on Rails</li>
+            <li class="list-items">RSPec</li>
+            <li class="list-items">CapyBara</li>
+            <li class="list-items">Selenium</li>
+          </ul>
+          <div class="heading-box">
+            <img
+              class="rectangular"
+              src="img/Rectangle2.png"
+              alt="round shape"
+            />
+            <h6 class="skills-headings">Skills</h6>
+          </div>
+          <ul>
+            <li class="list-items">Codekit</li>
+            <li class="list-items">Github</li>
+            <li class="list-items">Codeopen</li>
+            <li class="list-items">Gitlab</li>
```

Comment:
- [ ] Your portfolio is looking great! Let's make it even better by using `ul` and `li` tags also for the technology categories, e.g:

```
<ul>
    <li>Languages
        <ul>
            <li>JS</li>
            <li>HTML</li>
            <li>CSS</li>
        </ul>
    </li>    
    <li>Frameworks
        <ul>
            <li>Bootsrap</li>
            <li>RoR</li>
            <li>RSpec</li>
        </ul>
    </li>

     . . .   

</ul>
```